src/PopoverMenuItem.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { Link } from 'react-router';
+import ReactDOM from 'react-dom';
+import cx from 'classnames';
+
+/**
+ * @module PopoverMenuItem
+ */
+class PopoverMenuItem extends React.Component {
+
+	componentDidUpdate() {
+		if (this.selectedItem) {
+			ReactDOM.findDOMNode(this.selectedItem).focus();
+		}
+	}
```

Comment:
This setup can be simplified by checking for 'selected' status in one place rather than two, and since the `focus` call happens here, I recommend keeping a consistent reference to the `<Link>` instance with `ref={link => this.link = link}` in your `render` function. Then, in `componentDidUpdate`, you can check the props to determine whether or not to focus.

```js
componentDidUpdate() {
  if (this.props.isSelected) {
    ReactDOM.findDOMNode(this.link).focus();
  }
}
```